crates/cli-support/src/js/mod.rs

```diff
@@ -1415,8 +1415,8 @@ impl<'a> Context<'a> {
         self.expose_wasm_vector_len();
         self.global(&format!(
             "
-            function {}(arg, malloc) {{
-                const ptr = malloc(arg.length * {size}) >>> 0;
+            function {}(arg, malloc, align) {{
+                const ptr = malloc(arg.length * {size}, align) >>> 0;
```

Comment:
This doesn't really need to be an argument on the JS side - we already have access to the size of each element, which is the same as their alignment for all the types we support, so you can statically set it to `{size}` instead.

Reply:
Done.